src/main/java/de/doubleslash/keeptime/view/ViewController.java

```diff
@@ -156,6 +156,8 @@ private class Delta {
    private ReportController reportController;
 
    private Stage settingsStage;
+
+   private Stage confirmCloseStage;

```

Comment:
do we need this? does not seem to be used?

Reply:
deleted it